requirements_full.txt

```diff
@@ -0,0 +1,16 @@
+# Optional requirements - used by various plugins, features or tests
+fortran-language-server>=1.12.0,<1.13
+PyYAML>=5.4.1,<7
+# psycopg2-binary can lead to segfault - so far all seems good with v2.9.5
+psycopg2-binary==2.9.5
+pandas>=1.2,<2.1
+rdflib>=4.2.1,<7
+pint>=0.15,<1
+pymongo>=4.4.0,<5
+tripper>=0.2.5,<0.3
+requests>=2.10,<3
+jinja2>=3.0,<4
+#pydantic>=1.10.0,<2
```

Comment:
```suggestion
pydantic>=1.10.0,<2
```

Reply:
We support both Pydantic v1 and v2 now.

Reply:
This line should simply be removed then, or a comment should be noted about indirect dependencies in this file.